addons/sale_timesheet/models/project_update.py

```diff
@@ -73,15 +74,35 @@ def _get_profitability_values(self, project):
         if not (self.user_has_groups('project.group_project_manager') and costs_revenues):
             return {}
         profitability_items = project._get_profitability_items(False)
+        for revenue in profitability_items['revenues']['data']:
+            revenue['formatted'] = {
+                'invoiced': format_amount(self.env, revenue['invoiced'], project.currency_id),
+                'to_invoice': format_amount(self.env, revenue['to_invoice'], project.currency_id),
+                'expected_revenue': format_amount(self.env, revenue['invoiced'] + revenue['to_invoice'], project.currency_id),
+            }
+
+        for cost in profitability_items['costs']['data']:
+            cost['formatted'] = {
+                'billed': format_amount(self.env, cost['billed'], project.currency_id),
+                'to_bill': format_amount(self.env, cost['to_bill'], project.currency_id),
+                'expected_cost': format_amount(self.env, cost['billed'] - cost['to_bill'], project.currency_id),
+            }
         costs = sum(profitability_items['costs']['total'].values())
         revenues = sum(profitability_items['revenues']['total'].values())
         margin = revenues + costs
         return {
             'analytic_account_id': project.analytic_account_id,
             'costs': costs,
-            'costs_formatted': format_amount(self.env, -costs, project.currency_id),
+            'costs_formatted': {
+                'total_cost_billed': format_amount(self.env, profitability_items['costs']['total']['billed'], project.currency_id),
+                'total_cost_to_billed': format_amount(self.env, profitability_items['costs']['total']['to_bill'], project.currency_id),
+                'total_cost': format_amount(self.env, -costs, project.currency_id),},
```

Comment:
```suggestion
                'total_billed': format_amount(self.env, profitability_items['costs']['total']['billed'], project.currency_id),
                'total_to_billed': format_amount(self.env, profitability_items['costs']['total']['to_bill'], project.currency_id),
```
to be consistent with revenues

Reply:
Done the changes. Thanks :slightly_smiling_face: